src/plugins/saved_objects_management/public/lib/in_app_url.ts

```diff
@@ -38,6 +38,8 @@ export function canViewInApp(uiCapabilities: Capabilities, type: string): boolea
     case 'visualization':
     case 'visualizations':
       return uiCapabilities.visualize.show as boolean;
+    case 'augment-vis':
+      return uiCapabilities.visAugmenter.show as boolean;
```

Comment:
just curious, are we using this `show` for augment-vis? I recall it seems to be a registered capability.

Reply:
It's required for the inspection/edit to work. Otherwise there is an NPE since it tries to access capabilities for this new type, and nothing is set.